people.php

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<?php
+	$conn = require_once (__DIR__ . "/scripts/connection.php");
```

Comment:
```suggestion
	$conn = require (__DIR__ . "/scripts/connection.php");
```

Reply:
the file returns something, and does not define anything, so out of principle, `require` is better here